### DISCUSSION TITLE
Is it possible to import from GestioIP?

### DISCUSSION THREAD

**Discussion by fritzwoodtv** [netbox-community/devicetype-library#1802] [Q&A]
Is it possible to import from GestioIP?  Looking for a one sheet on how.  Thanks.

**Comment by danner26** [comment]
I am not sure, NetBox does have importing via YAML or CSV along with API access. I dont think there is a native integration.

That being said this might be better directed towards our [Slack channel](https://netdev.chat/) or the upstream [NetBox discussions](https://github.com/netbox-community/netbox/discussions). In the meantime I am going to close this as it is out of scope from this repo. Best of luck!
